projects/ebv-analyzer-app/src/app/ebv-selector/ebv-selector.component.ts

```diff
@@ -48,33 +42,22 @@ export class EbvSelectorComponent implements OnInit, OnDestroy {
     readonly containerDiv!: ElementRef<HTMLDivElement>;
 
     readonly isPlotButtonDisabled$: Observable<boolean>;
-    readonly loading$ = new BehaviorSubject(true);
 
-    ebvClasses: Array<EbvClass> = [];
-    ebvClass?: EbvClass = undefined;
-    ebvNames?: Array<string> = undefined;
-    ebvName?: string = undefined;
-    ebvDatasets?: Array<EbvDataset> = undefined;
-    ebvDataset?: EbvDataset = undefined;
+    rootCollectionId: ProviderLayerCollectionIdDict = {
+        providerId: '77d0bf11-986e-43f5-b11d-898321f1854c',
+        collectionId: '%22Classes%22',
```

Comment:
Huh, reminds me of someone: Okay, let's browse all "Classes".

Reply:
the collectionId is serialized Json. That's why the string has to have `"`s